src/components/credentials-popover-button/credentials-popover-button.component.css

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
the folder src/components/credentials-popover-button was removed in the main branch, it should not be added again